conventions/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -58,7 +58,7 @@ tasks.withType<JavaCompile>().configureEach {
           "-Xlint:-options",
 
           // Fail build on any warning
-          "-Werror"
+//          "-Werror"
```

Comment:
Same

Reply:
this means that we can't build with warnings and we can't allow this - I couldn't build the pubsub since there were some warnings here and there..